src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -192,7 +197,11 @@ class CollapsibleCourseRun extends React.Component {
               }
             />
           }
-          extraInput={{ onInvalid: this.openCollapsible }}
+          extraInput={{
+            onInvalid: this.openCollapsible,
+            min: 0,
+            max: 168,
```

Comment:
Why 168 for max out of curiosity?

Reply:
Yeah! "Min & Max Effort" ask for hours per week, and that's the max amount of hours in a week.